ibis/mapd/tests/conftest.py

```diff
@@ -50,3 +51,33 @@ def translate():
     dialect = MapDDialect()
     context = dialect.make_context()
     return lambda expr: dialect.translator(expr, context).get_result()
+
+
+def _random_identifier(suffix):
+    return '__ibis_test_{}_{}'.format(suffix, util.guid())
+
+
+@pytest.fixture
+def temp_table(con):
+    name = _random_identifier('table')
+    try:
+        yield name
+    finally:
+        assert con.exists_table(name), name
+        con.drop_table(name)
```

Comment:
@tonyfast not sure why but in the past I had some problems with drop function that broke the mapd server on CI